src/device/mod.rs

```diff
@@ -708,6 +712,36 @@ impl Device {
             .await?
         })
     }
+
+    /// Used by tcli only
+    #[cfg(any(target_os = "linux", target_os = "android"))]
+    pub fn probe_pmtu(&self, host: IpAddr) -> Result<u32> {
+        use std::os::fd::AsRawFd;
+
+        self.art()?.block_on(async {
+            let sock = telio_pmtu::PMTUSocket::new(host)?;
+            telio_log_debug!("PMTU socket created");
+
+            match self.rt() {
+                Ok(rt) => {
+                    task_exec!(rt, async move |rt| {
+                        let future = async {
+                            telio_log_debug!("Making PMTU socket external");
+                            rt.entities.socket_pool.make_external(sock.as_raw_fd());
+                            sock.probe_pmtu().await
+                        };
+
+                        Ok(future.await.map_err(Error::PmtuProbe))
+                    })
+                    .await?
+                }
+                Err(_) => {
+                    // The device is not started. Let us create runtime then
```

Comment:
This call is not exposed to ffi layer but is only used by tcli in nat-lab. Erroring out would break the nat-lab tests for PMTU. Besides what's wrong with creating ephemeral runtime here?

Reply:
Oh, I'm sorry. It was created here but probably was lost due to constant rebases. I'll fix this

Reply:
Fixed it. It was simply a leftover because the runtime is created on device creation.